packages/flutter/lib/src/material/flat_button.dart

```diff
@@ -149,4 +149,25 @@ class FlatButton extends StatelessWidget {
       child: child
     );
   }
+
+  @override
+  void debugFillDescription(List<String> description) {
+    super.debugFillDescription(description);
+
+    if (onPressed != null)
```

Comment:
Actually I'm not sure this is a correct way 🙇 
I would appreciate if you tell the better way.